views/pages/addCustom.ejs

```diff
@@ -30,12 +30,12 @@
 
         <!--Input product lifetime-->
         <label for="name">Product Lifetime (days):</label>
-        <input type="number" id="prod_life"/> 
+        <input type="number" onkeypress="return event.charCode >= 48" min="1" id="prod_life"/> 
```

Comment:
What does the onkeypress event do in this case? Is it supposed to be ensuring that only numbers can be added for these fields?

Reply:
If this is the case, that's a cool way to do the validation, but you should be aware that most keycodes are actually above 48, including all letters and most symbols, while the range for numbers 0 through 9 is specifically 48 - 57 (inclusive). So to get this to work for only numbers, you should change this to include both bounds, such as:

```onkeypress="return event.charCode >= 48 && event.charCode <= 57" min="1" ```

Reply:
Thanks.  Fixed!